.github/workflows/benchmarks.yml

```diff
@@ -28,7 +28,7 @@ concurrency:
 jobs:
   benchmarks:
     runs-on: ubuntu-22.04
-    if: github.repository == 'GenericMappingTools/pygmt'
+    if: github.repository == 'GenericMappingTools/pygmt' && contains(github.event.pull_request.labels.*.name, 'run/benchmark')
```

Comment:
Actually I'm unsure if the workflow will run in the main branch, because it's a push event not a pull_request event.

Reply:
Tried in my own fork. It doesn't work in the main branch.

Reply:
Changed the `if` condition to:
```
if: github.repository == 'GenericMappingTools/pygmt' && (github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'run/benchmark'))
```
Now it works in my own fork, see the changes between my fork and the pygmt repository: https://github.com/GenericMappingTools/pygmt/compare/main...seisman:pygmt:main and the workflow was triggered when pushing in the main branch (https://github.com/seisman/pygmt/actions/workflows/benchmarks.yml).